test/groovy/util/JenkinsReadYamlRule.groovy

```diff
@@ -25,11 +25,14 @@ class JenkinsReadYamlRule implements TestRule {
         return new Statement() {
             @Override
             void evaluate() throws Throwable {
-                testInstance.helper.registerAllowedMethod("readYaml", [Map], { Map m ->
```

Comment:
keeping the "m" would shrink the diff, which in turn would un-blur the diff.

Reply:
m is meaning less

Reply:
My opinion is that 'i' for loops is standard, 'm' not. Could you please review first PR#71, then after I can rebase this PR and the diff will be smaller.

Reply:
@OliverNocon @CCFenner 
I would like to get more opinions from the developer community.
What about naming of variables used on closures/lambdas

[ ] we should try to have speaking names
[ ] short names like m for Maps, c for Closures are fine
[ ]  there is no need to aggree on a naming pattern
[ ] ...

Reply:
- [x] depends on my current state of mind..

I would say short names are ok, if it's only one parameter.

Reply:
https://github.com/SAP/jenkins-library/blob/master/CONTRIBUTING.md#code-style

"Variables, methods, types and so on shall have meaningful self describing names. Doing so makes understanding code easier and requires less commenting. It helps people who did not write the code to understand it better."

And it was not written by me :)